app/models/transaction.rb

```diff
@@ -0,0 +1,9 @@
+class Transaction < ApplicationRecord
+  belongs_to :user
+
+  validates :sum, :date, presence: true
+  validates :sum, numericality: true
+
+  scope :most_recent, ->(user) { where(user_id: user).order(created_at: :desc).limit(10) }
+  scope :total, ->(user) { sum(:sum).where(user_id: user) }
```

Comment:
@sinpai you've added two scopes here, but I don't see where they used in the PR. If you are planning to use them later - let's add the later.

Reply:
@sinpai after all, we have to have tests for those scopes. they can be located in `rspec/unit/models/user_spec.rb` file

Reply:
FYI: `spec/models/transaction_spec.rb`

Reply:
@AleksSenkou 👍 I've removed those scopes from this PR. I will add it in PRs, when it's required to use them (along with tests).